backend/src/main/resources/application.properties

```diff
@@ -0,0 +1,17 @@
+spring.application.name=backend
+# This is the property to specify the database and the driver here todolist is database name and mysql is driver
+spring.datasource.url=jdbc:mysql://localhost:3306/skyrise
+
+# These properties specify the data used for authentication and authorization for the database
+spring.datasource.username=
```

Comment:
Return the user name and password

spring.datasource.username= skyrise_admin
spring.datasource.password= skyrise_app_swe26